pallets/weights/src/pallet_asset.rs

```diff
@@ -491,4 +491,16 @@ impl pallet_asset::WeightInfo for SubstrateWeight {
             .saturating_add(DbWeight::get().reads(18))
             .saturating_add(DbWeight::get().writes(9))
     }
+    fn exempt_ticker_affirmation() -> Weight {
+        unimplemented!();
+    }
+    fn remove_ticker_affirmation_exemption() -> Weight {
+        unimplemented!();
+    }
+    fn pre_approve_ticker() -> Weight {
+        unimplemented!();
+    }
+    fn remove_ticker_pre_approval() -> Weight {
+        unimplemented!();
```

Comment:
Please fill these in.  You can use the dev server to run the benchmarks if you don't want to run them locally.